### PR TITLE
[feat] Add Image

### PR DESCRIPTION
**Pull request by Ronak-Mutha**
# Image
- responsive image
image

- round image
image

### CODE REVIEW

**Comment by hinanshisuthar** [comment]
Great work done, Ronak. Good to merge.

**Review by hinanshisuthar — APPROVED** [review]
Great work done, Ronak. Good to merge.

File: docs/image.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                    <ul class="nav-links">
+                        <li class="m-2">
+                            <a href="../index.html" class="br-2">Home</a>
+                        </li>
+                        <li class="m-2 active">
+                            <a href="/docs/installation.html" class="br-2">Docs</a>
+                        </li>
+                        <li class="m-2">
+                            <a href="https://github.com/Ronak-Mutha/unstoppable-ui" target="_blank"
+                                rel="noopener noreferrer" class="br-2">Contribute <i
```
Comment (saadahmadhere) [inline docs/image.html:42]: <i> tag should be opened in the next line

**Review by shivampandey0 — APPROVED** [review]
Structured HTML with the use of utility classes.
Good to merge

**Review by saadahmadhere — APPROVED** [review]
Beautifully written code. Hats off to the work you did in such a short span.

File: docs/image.html (hunk 1 above)
Comment (Ronak-Mutha) [reply docs/image.html:42]: Thanks for pointing this out Saad! I'll definitely check and configure prettier again, so it works as intended and re-run prettier on whole code once projects are completed.

**Comment by Ronak-Mutha** [comment]
Got 2 reviews. Merging to dev branch and closing this branch.

**Pull request merged**